yosh/builtins/history.py

```diff
@@ -0,0 +1,19 @@
+import os
+from yosh.constants import *
+
+
+def history(args):
+    count = 0
+    if args:
```

Comment:
Use `if len(args) > 0:` to be more explicit what is being checked. (If we use only `if args`, then, if args is a string or any number not equal 0, it evaluates to be true.